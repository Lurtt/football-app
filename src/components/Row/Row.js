import styled from 'styled-components'

import { justifyContent } from '../mixins'

export default styled.div`
  display: flex;
  ${justifyContent};
`
