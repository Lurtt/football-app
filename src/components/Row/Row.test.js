import React from 'react'
import renderer from 'react-test-renderer'

import Row from './'

describe('<Row />', () => {
  test('Should render', () => {
    const result = renderer
      .create(<Row justifyContent="center">Content here</Row>)
      .toJSON()
    expect(result).toMatchSnapshot()
  })
})
