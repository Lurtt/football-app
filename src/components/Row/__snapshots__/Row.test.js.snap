// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Row /> Should render 1`] = `
<div
  className="sc-bdVaJa hYebyD"
>
  Content here
</div>
`;
